src/settings-ui/Settings.UI/Strings/en-us/Resources.resw

```diff
@@ -1884,48 +1884,53 @@ From there, simply click on a Markdown file, PDF file or SVG icon in the File Ex
     <value>Example: outlook.exe</value>
   </data>
   <data name="AlwaysOnTop_FrameColor.Header" xml:space="preserve">
-    <value>Frame color</value>
+    <value>Color</value>
   </data>
-  <data name="AlwaysOnTop_FrameEnabled.Description" xml:space="preserve">
-    <value>Draw frame around pinned window</value>
+	<data name="AlwaysOnTop_FrameEnabled.Header" xml:space="preserve">
+    <value>Border</value>
   </data>
-  <data name="AlwaysOnTop_FrameEnabled.Header" xml:space="preserve">
-    <value>Frame</value>
+  <data name="AlwaysOnTop_FrameEnabled.Description" xml:space="preserve">
+    <value>Show a border around the pinned window</value>
   </data>
   <data name="AlwaysOnTop_FrameThickness.Header" xml:space="preserve">
-    <value>Frame thickness</value>
+    <value>Thickness</value>
   </data>
   <data name="AlwaysOnTop_Behavior_GroupSettings.Header" xml:space="preserve">
-    <value>Behavior</value>
+   <value>Appearance &amp; behavior</value>
   </data>
   <data name="Shell_AlwaysOnTop.Content" xml:space="preserve">
     <value>Always On Top</value>
   </data>
-  <data name="AlwaysOnTop_GameMode.Header" xml:space="preserve">
-    <value>Game Mode</value>
-  </data>
-  <data name="AlwaysOnTop_PreventActivationOnGameMode.Content" xml:space="preserve">
+  <data name="AlwaysOnTop_GameMode.Content" xml:space="preserve">
     <value>Do not activate when Game Mode is on</value>
+	  <comment>Game Mode is a Windows feature</comment>
+  </data>
+  <data name="AlwaysOnTop_Sound.Content" xml:space="preserve">
+    <value>Play a sound when (un)pinning a window</value>
```

Comment:
```suggestion
    <value>Play a sound when pinning and unpinning a window</value>
```

Easier to translate into other langs and easier to read when looking fast.

Reply:
Just leave off unpinning.